### ISSUE TITLE
![image](https://user-images.githubusercontent.com/89205116/165457452-6be462c7-f983-48f3-97c9-b06288af4446.png)

### ISSUE THREAD

**Issue by DeltaJuice** [Solar-Tweaks/Solar-Tweaks#17]
![image](https://user-images.githubusercontent.com/89205116/165457452-6be462c7-f983-48f3-97c9-b06288af4446.png)
for me the files generated but it didnt update patcher in game|
For example 
![image](https://user-images.githubusercontent.com/89205116/165457615-b83f9725-1b6a-46b6-8ccb-f3340dfeb9cf.png)
here fps is changed to cps and its enabled but in game its still normal
![image](https://user-images.githubusercontent.com/89205116/165457781-1a07e1c8-9459-4300-8127-d45c5d8b985b.png)
few more tweaks haven't patched in game even though they are selected

_Originally posted by @DeltaJuice in https://github.com/Solar-Tweaks/Solar-Tweaks/issues/7#issuecomment-1110607511_

**Comment by 770grappenmaker** [comment]
Okay, so what i’m imagining is that you somehow managed to break the config (i saw that happen to someone else as well). Unfortunately there isn’t a good way to debug just that (well there is but not that easy for normal users). Could you send me the config? Also, try to enable checks again, so disable the “skip checks” box.

**Comment by DeltaJuice** [comment]
ok how do i upload my config(new to github commenting))

**Comment by DeltaJuice** [comment]
I disabled checks but still the same issue is present

**Comment by 770grappenmaker** [comment]
![image](https://user-images.githubusercontent.com/50543283/165470656-71aabead-e896-43f3-bb00-a87d73b60ae0.jpeg)

**Comment by DeltaJuice** [comment]
it says i cant upload jsons

**Comment by DeltaJuice** [comment]
ill just copy paste

**Comment by DeltaJuice** [comment]
{
  "patcherVersion": "1.6",
  "debug": false,
  "enableAll": false,
  "nickhider": {
    "from": "You",
    "to": "You",
    "isEnabled": false
  },
  "fps": {
    "pattern": "\u0001 %s",
    "from": "FPS",
    "to": "cps",
    "isEnabled": true
  },
  "cps": {
    "pattern": "\u0001 %s",
    "from": "CPS",
    "to": "",
    "isEnabled": false
  },
  "autoGG": {
    "from": "/achat gg",
    "to": "/ac GG",
    "isEnabled": true
  },
  "levelHead": {
    "pattern": "%s: ",
    "from": "Level",
    "to": "IQ",
    "isEnabled": true
  },
  "metadata": {
    "removeCalls": [
      "modSettings",
      "pinnedServers",
      "blogPosts"
    ],
    "isEnabled": true
  },
  "modpacketRemoval": {
    "isEnabled": false
  },
  "cloaksPlus": {
    "isEnabled": false
  },
  "windowName": {
    "from": "Lunar Client (\u0001-\u0001/\u0001)",
    "to": "Lunar Client (Modded by Solar Tweaks)",
    "isEnabled": true
  },
  "noHitDelay": {
    "isEnabled": true
  },
  "fpsSpoof": {
    "multiplier": 2,
    "isEnabled": false
  },
  "customCommands": {
    "commands": {
      "qb": {
        "prefix": "/play duels_bridge_duel",
        "suffix": ""
      },
      "qbd": {
        "prefix": "/play duels_bridge_doubles",
        "suffix": ""
      },
      "db": {
        "prefix": "/duel ",
        "suffix": " bridge"
      },
      "bwp": {
        "prefix": "/play bedwars_practice",
        "suffix": ""
      }
    }
  },
  "rpcUpdate": {
    "clientID": "Delta#9920",
    "icon": "logo",
    "iconText": "KoolKatKlient",
    "afkText": "AFK",
    "menuText": "In Menu",
    "singlePlayerText": "Alone sed",
    "versionText": "Minecraft",
    "displayActivity": true,
    "customServerMappings": {},
    "isEnabled": true
  },
  "tasklistPrivacy": {
    "isEnabled": false,
    "enabled": true
  },
  "hostslistPrivacy": {
    "isEnabled": false,
    "enabled": true
  },
  "uncapReach": {
    "isEnabled": false
  },
  "removeFakeLevelhead": {
    "isEnabled": true
  },
  "removeHashing": {
    "className": "WMMWMWMMWMWWMWMWWWWWWWMMM",
    "method": {
      "name": "WWWWWNNNNWMWWWMWWMMMWMMWM",
      "descriptor": "(Ljava/lang/String;[BZ)Z",
      "owner": null,
      "access": null
    },
    "isEnabled": false
  },
  "debugPackets": {
    "isEnabled": false
  },
  "toggleSprintText": {
    "replacements": {
      "flying": "flying",
      "riding": "riding",
      "descending": "descending",
      "dismounting": "dismounting",
      "sneaking": "sneaking",
      "sprinting": "sprinting"
    },
    "method": {
      "name": "<clinit>",
      "descriptor": "()V",
      "owner": null,
      "access": 8
    },
    "isEnabled": true,
    "flying": "Flight",
    "riding": "Riding",
    "descending": "Down",
    "dismounting": "Dismounting",
    "sneaking": "Scaffold",
    "sprinting": "KillAura"
  },
  "reachText": {
    "pattern": "\u0001 %s",
    "from": "blocks",
    "to": "blocks",
    "isEnabled": false
  },
  "fixPings": {
    "isEnabled": false
  },
  "lunarOptions": {
    "isEnabled": true
  },
  "supportOverlays": {
    "isEnabled": true
  },
  "toggleSneakContainer": {
    "isEnabled": false
  },
  "pingText": {
    "pattern": "\u0001 %s",
    "from": "ms",
    "to": "ms",
    "isEnabled": true
  },
  "pingSpoof": {
    "pingValue": "100",
    "isEnabled": false
  },
  "clothCapes": {
    "isEnabled": false
  },
  "hurtCamShake": {
    "multiplier": 0.3,
    "isEnabled": true
  },
  "chatLimit": {
    "limit": 255,
    "isEnabled": false
  },
  "mumbleFix": {
    "isEnabled": false
  },
  "websocket": {
    "to": "wss://assetserver.lunarclientprod.com/connect",
    "isEnabled": false
  },
  "enableWrapped": {
    "isEnabled": false
  },
  "customMods": {
    "textMods": []
  }
}

**Comment by 770grappenmaker** [comment]
That should be working, do other modules work for you?

**Comment by DeltaJuice** [comment]
the pack thing works i am able to apply custom lunar textures

**Comment by 770grappenmaker** [comment]
yeah thats because it is in the default config enabled

**Comment by 770grappenmaker** [comment]
![image](https://user-images.githubusercontent.com/50543283/165489877-b3f2fc3c-cf79-495f-bedd-1b8a44f78e62.png)
That doesn't work either, by the way. Try resetting it to `920998351430901790`

**Comment by DeltaJuice** [comment]
k ty ill reset and try again

**Comment by DeltaJuice** [comment]
the fps text is changed but the sprint toggled texts are not

**Comment by 770grappenmaker** [comment]
Did you enable it?

**Comment by DeltaJuice** [comment]
yes

**Comment by 770grappenmaker** [comment]
What is your config like now?

**Comment by DeltaJuice** [comment]
basically the same

**Comment by 770grappenmaker** [comment]
![image](https://user-images.githubusercontent.com/50543283/165589961-0c547eed-8805-4370-9890-cb6f8d5427ca.png)
But... you didn't change it... so why should it be any different?

**Comment by UzielHDD** [comment]
i have a similar problem, i have the patcher enabled and the files generate but i dont get the changes in game

**Comment by 770grappenmaker** [comment]
Please create a seperate discussion.

**Comment by 770grappenmaker** [comment]
Did you enable "skip checks" (you should not)

**Comment by UzielHDD** [comment]
no i didnt

**Comment by 770grappenmaker** [comment]
I'm pretty sure this is not a patcher issue but rather a launcher issue, or your configuration is invalid. Can you verify this by running /solardebug ingame? (And then attaching a screenshot)

**Comment by UzielHDD** [comment]
![image](https://user-images.githubusercontent.com/89594267/165738711-89a25f3f-3598-482f-bdf0-23c299ee8874.png)

**Comment by 770grappenmaker** [comment]
Yep, exactly what i thought. Your config must be invalid. Try resetting it by deleting it, and also the file in %APPDATA%\solartweaks\settings.json

**Comment by UzielHDD** [comment]
still doesnt work

**Comment by 770grappenmaker** [comment]
Bruh you need to reconfigure through the launcher now that you reset it...

**Comment by UzielHDD** [comment]
i did, it still doesnt work

**Comment by 770grappenmaker** [comment]
You deleted both config.json and settings.json right?

**Comment by UzielHDD** [comment]
fuck, i didnt, now it works thanks my bad

**Comment by novemmm** [comment]
> @DeltaJuice Are you still having issues? If so, please reopen (or preferably, make a discussions page, issues aren't for support)

help ihave the same problem the mods in patcher arent working how do i delete the config.json file?

**Comment by novemmm** [comment]
also there is nothing in the file called solar-patcher.jar and the command solar debug doesnt work

**Comment by novemmm** [comment]
/solardebug doesnt work no patcher mod worked
